incubator/hnc/test/e2e/quickstart_test.go

```diff
@@ -249,4 +249,39 @@ spec:
 		RunShouldContain(expected, defTimeout, "kubectl hns tree", nsTeamA)
 		RunShouldContain("ActivitiesHalted (ParentMissing):", defTimeout, "kubectl hns describe", nsStaging)
 	})
+
+	It("Should demonstrate exceptions", func() {
+		// set up initial structure
+		CreateNamespace(nsOrg)
+		CreateSubnamespace(nsTeamA, nsOrg)
+		CreateSubnamespace(nsTeamB, nsOrg)
+
+		MustRun("kubectl -n", nsOrg, "create secret generic my-secret --from-literal=password=iamacme")
+		// allow secret to propagate
+		MustRun("kubectl hns config set-resource secrets --mode Propagate --force")
+		// wait 2 seconds to give time for secret to propogate
+		time.Sleep(2 * time.Second)
+		// check that the secrete has been propagated to both subnamespaces
+		RunShouldContain("my-secret", defTimeout, "kubectl -n", nsTeamA, "get secrets")
+		RunShouldContain("my-secret", defTimeout, "kubectl -n", nsTeamB, "get secrets")
```

Comment:
If the `sleep` above is because the wait time is long, you may use a different time out here instead of `defTimeout` and remove the `sleep`.

Reply:
removed the sleep above